pkg/sql/source.go

```diff
@@ -19,6 +19,8 @@ WHERE (i.indisprimary OR i.indisunique) AND i.indisvalid AND i.indpred IS NULL O
 
 var CreateLogicalSlot = `SELECT pg_create_logical_replication_slot($1, $2);`
 
+var CreatePublication = `CREATE PUBLICATION %s FOR ALL TABLES;`
```

Comment:
Why do we need this? Isn't this meant to be used with `CREATE SUBSCRIPTION`?

Reply:
publication is more like pglogical local node provider.
Each publication can be matched a replication slot and a subscription, but we only need publication and replication slot.
When starting replication, we should use pgoutput plugin and need to give publication names as plugin parameters.